source/administration/devops-command-center.rst

```diff
@@ -0,0 +1,284 @@
+
+The creator of an incident automatically becomes the incident Commander, responsible for managing the incident.
+
+The Commander of the incident can:
+
+- Select Playbooks
+- Create checklists
+- Invite people to the channel
+- Hand over the Commander role to another channel member
+
+The Commander of the channel must be a team/channel admin, and if the role is handed to another member they must also be a team/channel admin. To change Commanders,
+select the current Commander’s name and use the search bar to locate the username of the new Commander. Only members of the channel may be selected as commanders. To assign a
+Commander not in the team, that user must first be invited to the channel using the existing Mattermost user interface (**Manage Members** in the channel header, or the /invite slash command)
+Changing the Commander takes effect immediately, and members are notified of the change in the channel.
+
+Active incidents
+~~~~~~~~~~~~~~~~
+
+To view details about active incidents, click the **!** icon in the channel header to bring up the right-hand side (RHS) panel that shows a list of current
+active incidents. Select an incident to see its Commander, channel, and checklist.
+
+When an incident has started and the incident channel is created, the channel header is updated to reflect its origin and
+a message from the incident bot is also posted to the incident channel naming the creator of the incident. If an incident is started
+from the context of a post, the channel header includes a link to the post and the text of the post is also posted to the channel alongside a permalink.
+
+Incidents are displayed in the RHS and visible to all members of the relevant team, even if they are not a member of the corresponding incident channel. It
+is not possible to view incidents from teams other than the currently selected team.
+
+Using Playbooks
+~~~~~~~~~~~~~~~~
+
+Playbooks are templates which are applied to an incident and define a set of steps to be followed in order to resolve the incident. Playbooks can be
+refined over a period of time to match the changing parameters of related incidents.
+
+The set of steps contained within a Playbook is called a checklist.
+
+Viewing and creating Playbooks
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Select the Playbooks icon **()** in the RHS to open the Playbooks backstage and list all Playbooks associated with the current team.
+Select **+ New Playbook** to start a new Playbook and checklist. Enter a Playbook name and choose **Save Playbook**.
+
+Creating checklists
+~~~~~~~~~~~~~~~~~~~
+
+Checklists are created within a Playbook and provide steps to follow during an incident. Checklists can be created ahead of time as part of a Playbook, or during an incident.
+
+
+Editing checklists
+~~~~~~~~~~~~~~~~~~~
+
+Checklists can be edited during an incident as well as when not associated with an active incident.
```

Comment:
I did not understand well the last part of this sentence. Do you mean checklists in playbooks?